Can't resolve lodash.merge

I'm getting the following error upon importing, with the basic auto-tracking config:

```
ERROR in ./~/vue-analytics/dist/vue-analytics.js
Module not found: Error: Can't resolve 'lodash.merge' in '...\node_modules\vue-analytics\dist'
 @ ./~/vue-analytics/dist/vue-analytics.js 7:90-113
```

--- a/package.json
+++ b/package.json
@@ -52,11 +52,12 @@
     "eslint-loader": "^1.6.0",
     "eslint-plugin-promise": "^3.3.0",
     "eslint-plugin-standard": "^2.0.1",
-    "lodash.merge": "^4.6.0",
     "rollup": "^0.39.2",
     "rollup-plugin-babel": "^2.7.1",
     "rollup-plugin-uglify": "^1.0.1",
     "webpack": "^1.13.2"
   },
-  "dependencies": {}
+  "dependencies": {
+    "lodash.merge": "^4.6.0"
+  }
 }

